wishlist.html

```diff
@@ -0,0 +1,202 @@
+          <div class="badge-count">4</div>
+        </figure>
+
+        <figure class="icon icon-notification ">
+          <i class="fa-solid fa-bell"></i>
+          <div class="badge-count">1</div>
+        </figure>
+      </div>
+    </div>
+  </div>
+  <div class="body">
+    <div class="wishlist-container col-4-layout">
+      <div class="col-4-4">
+
+        <h2 class="align-center">Wishlist</h2>
+        <div class="wishlist-grid">
+          <div class="product">
+            <div class="card product-card">
+              <div class="card-image-with-badge">
+                <img class="card-image" src="./images/shoe.png" alt="" />
+                <figure>
+                  <i class="fa-solid fa-heart"></i>
+                </figure>
+              </div>
+              <div class="card-description">
+                <h2>Converse</h2>
+                <p>
+                  Men High Top Shoes
+                </p>
+                <p>
+                  ₹1299 <strike class="card-strk">₹1599</strike>
+                </p>
+                <div class="card-buttons">
+                  <button class="button button-icon wishlist-button">
+                    Remove From Cart
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="product">
+            <div class="card product-card">
+              <div class="card-image-with-badge">
+                <img class="card-image" src="./images/shoe.png" alt="" />
+                <figure>
+                  <i class="fa-solid fa-heart"></i>
+                </figure>
+              </div>
+              <div class="card-description">
+                <h2>Converse</h2>
+                <p>
+                  Men High Top Shoes
+                </p>
+                <p>
+                  ₹1299 <strike class="card-strk">₹1599</strike>
+                </p>
+                <div class="card-buttons">
+                  <button class="button button-icon wishlist-button">
+                    Remove From Cart
+                  </button>
+
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="product">
+            <div class="card product-card">
+              <div class="card-image-with-badge">
+                <img class="card-image" src="./images/shoe.png" alt="" />
+                <figure>
+                  <i class="fa-solid fa-heart"></i>
+                </figure>
+              </div>
+              <div class="card-description">
+                <h2>Converse</h2>
+                <p>
+                  Men High Top Shoes
+                </p>
+                <p>
+                  ₹1299 <strike class="card-strk">₹1599</strike>
+                </p>
+                <div class="card-buttons">
+                  <button class="button button-icon wishlist-button">
+                    Remove From Cart
+                  </button>
```

Comment:
If we are on the wishlist page , then how can you remove that item from cart . Replace with Add to cart , Rest looks good .  work on your responsiveness though.